src/tracer/wrappers/OMP/gnu-libgomp.c

```diff
@@ -2765,7 +2839,20 @@ static int gnu_libgomp_get_hook_points (int rank)
 	int count = 0;
 
 	UNREFERENCED_PARAMETER(rank);
+#if defined (OS_RTEMS)
```

Comment:
This could be moved just before the else for dlsyms.

Reply:
Agree